baseframe/static/js/baseframe.js

```diff
@@ -1,10 +1,6 @@
 // This is a global function. Isn't there a better way to do this?
 
 function activate_widgets() {
-  // Activate select2.js and CodeMirror for non-mobile browsers
-  if (!navigator.userAgent.match(/(iPod|iPad|iPhone|Android)/)) {
-    $('select:not(.notselect)').select2({allowClear: true});
-
```

Comment:
Now you've activated Select2 on mobile as well, which is not wanted. Only move CodeMirror out.